src/test/java/com/hazelcast/jclouds/ComputeServiceBuilderInvalidConfigTest.java

```diff
@@ -22,38 +21,38 @@
 @Category(QuickTest.class)
 public class ComputeServiceBuilderInvalidConfigTest extends HazelcastTestSupport {
 
-    public static final String configTemplate = "<hazelcast xsi:schemaLocation=\"http://www.hazelcast.com/schema/config hazelcast-config-3.6.xsd\"" +
-            "           xmlns=\"http://www.hazelcast.com/schema/config\"" +
-            "           xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\">" +
-            "    <properties>" +
-            "        <property name=\"hazelcast.discovery.enabled\">true</property>" +
-            "    </properties>" +
-            "    <network>" +
-            "        <join>" +
-            "            <multicast enabled=\"false\"/>" +
-            "            <tcp-ip enabled=\"false\" />" +
-            "            <aws enabled=\"false\"/>" +
-            "            <discovery-strategies>" +
-            "                <discovery-strategy enabled=\"true\" class=\"com.hazelcast.jclouds.JCloudsDiscoveryStrategy\">" +
-            "                    <properties>" +
-                                 "${PROPERTIES_PLACE_HOLDER}"+
-            "                    </properties>" +
-            "                </discovery-strategy>" +
-            "            </discovery-strategies>" +
-            "        </join>" +
-            "    </network>" +
-            "</hazelcast>";
+    private static final String configTemplate
+            = "<hazelcast xsi:schemaLocation=\"http://www.hazelcast.com/schema/config hazelcast-config-3.6.xsd\""
```

Comment:
Shouldn't we bump up this to 3.7 (as we support 3.7+)?

Reply:
As we upgraded hazelcast dependency to 3.9, we can set xsd version directly to 3.9

Reply:
Good catch, fixed.